src/main/java/mate/academy/service/impl/UserServiceImpl.java

```diff
@@ -18,7 +19,13 @@ public Optional<User> findByEmail(String email) {
     }
 
     @Override
-    public User add(User user) {
```

Comment:
method signature should have still been the same as it was defined in the task description, this user instance would just be set a salt and a new password, user object itself should still be created in the `register()` method